### ISSUE TITLE
More helm default overrides

### ISSUE THREAD

**Issue by gorzek** [Praqma/helmsman#55]
Currently, helmsman allows overriding of a few helm defaults, such as adding the "wait" flag on a given app.

It would be useful to have most or all of the command-line options of helm represented in helmsman, so it is possible to do things like extend the timeout (--timeout flag).

**Comment by akshayks** [comment]
On a related note, it would be nice to be able to specify overrides for `helm init` to customize where `tiller` is installed. For example, I would like to restrict `tiller` to a specific set of nodes allocated for Kubernetes controllers and the like. When manually installing `tiller` into our cluster I employ a command similar to:
```
helm init --service-account tiller
--override "spec.template.spec.affinity.nodeAffinity.requiredDuringSchedulingIgnoredDuringExecution.nodeSelectorTerms[0].matchExpressions[0].key"="role"
--override "spec.template.spec.affinity.nodeAffinity.requiredDuringSchedulingIgnoredDuringExecution.nodeSelectorTerms[0].matchExpressions[0].operator"="In"
--override "spec.template.spec.affinity.nodeAffinity.requiredDuringSchedulingIgnoredDuringExecution.nodeSelectorTerms[0].matchExpressions[0].values[0]"="kube-system"
--override "spec.template.spec.tolerations[0].key"="KubeSystemOnly"
--override "spec.template.spec.tolerations[0].operator"="Exists"
--override "spec.template.spec.tolerations[0].effect"="NoSchedule"
```
It is currently not possible to configure this via `helmsman`.

I'm very much a go newbie but managed to modify `helmsman` code to allow injecting overrides. I would be happy to share what I have via a pull request (or other means) if anyone is interested.

**Comment by sami-alajrami** [comment]
@akshayks feel free to create a pull request with your changes :)

**Comment by sami-alajrami** [comment]
@gorzek I am adding `timeout` & `no-hooks` for install/upgrade. Let me know if you have other options in mind.

**Comment by sami-alajrami** [comment]
@akshayks yes :)

**Comment by sami-alajrami** [comment]
this is now shipped in v1.5.0. I will close this, but feel free to reopen or create new issue if you need more helm options.
